vas/task_1/src/app/app-routing.module.ts

```diff
@@ -0,0 +1,27 @@
+import { NgModule } from '@angular/core';
+import { RouterModule, Routes } from '@angular/router';
+import { PageNotFoundComponent } from './page-not-found/page-not-found.component';
+
+const routes: Routes = [
+  {
+    path: 'home',
+    loadChildren: () => import('./home/home-routing.module').then(mod => mod.HomeRoutingModule)
```

Comment:
as per Angular documentation https://angular.io/guide/lazy-loading-ngmodules#lazy-loading-basics

```suggestion
    loadChildren: () => import('./home/home.module').then(mod => mod.HomeModule)
```

should be used instead.